example/rails-51/README.md

```diff
@@ -64,10 +70,17 @@ Once PostgreSQL is set up as detailed using [the PostgreSQL documentation](https
 
 ### Configuration
 
+All tasks run with Que should set the rails environment to `que`.  This ensures that the correct database and connection settings are used.
+Do this by prepending `RAILS_ENV=que` before each command, or run:
+
+```shell
+export RAILS_ENV=que
+```
+
 Ensure that the initial Que setup is complete by running:
 
 ```shell
-bundle exec bin/rake que:install
+bundle exec bin/rails generate que:install
```

Comment:
In this example app, running db:migrate on the migration generated by running this command fails for me:
```
bugsnag-ruby/example/rails-51$ RAILS_ENV=que be rake db:migrate
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:

Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:

  class AddQue < ActiveRecord::Migration[4.2]
```

We might need que 0.14.x for it to work (unless I'm doing something wrong).

Reply:
I get the same...